pages/sponsors.md

```diff
@@ -14,7 +14,7 @@ JHipster uses [OpenCollective](https://opencollective.com/generator-jhipster) to
 
 - Visibility on the front page of our website for all sponsors and backers (about 100,000 views/month at the end of 2017)
 - Visibility on the Readme file of the main project for all sponsors and backers (about 60,000 views/month at the end of 2017)
-- Company logos on all [http://www.jhipster.tech](http://www.jhipster.tech) pages for gold sponsors (about 450,000 views/month at the end of 2017)
+- Company logos (or) link to your websites on all [http://www.jhipster.tech](http://www.jhipster.tech) pages for gold sponsors (about 450,000 views/month at the end of 2017)
```

Comment:
I'll modify the "or", as there should be a link on the logo

Reply:
argh this is done by OpenCollective so it will link to their company Opencollective page. That's complex to explain, I'll just keep logo

Reply:
No it is doable, checkout webpack again they show the person's photo with a link to their website

Reply:
OK I'll have a look, but then it will be "logos AND link" so this needed to be changed anyway